godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -152,14 +155,13 @@ class TractViewController: BaseViewController {
     
     // MARK: - Segmented Control
     
-    fileprivate func languageSegmentedControl() -> UISegmentedControl {
+    fileprivate func configureLanguageSegmentedControl() {
         let primaryLabel = self.determinePrimaryLabel()
         let parallelLabel = self.determineParallelLabel()
         
-        let segmentedControl = UISegmentedControl(items: [primaryLabel, parallelLabel])
-        segmentedControl.selectedSegmentIndex = 0
-        segmentedControl.addTarget(self, action: #selector(didSelectLanguage), for: .valueChanged)
-        return segmentedControl
+        languageSegmentedControl = UISegmentedControl(items: [primaryLabel, parallelLabel])
+        languageSegmentedControl!.selectedSegmentIndex = 0
```

Comment:
The app is crashing in this line because `languageSegmentedControl` is nil, so it is not retaining the class, the `weak` is causing this (don't know why, it should not happen), but, do we really need the `weak` for this?